src/profile-logic/profile-data.js

```diff
@@ -1530,12 +1566,20 @@ export function hasUsefulSamples(
  * This function takes both a SamplesTable and can be used on CounterSamplesTable.
  */
 export function getSampleIndexRangeForSelection(
-  table: { time: Milliseconds[], length: number },
+  times: { time: Milliseconds[], length: number },
```

Comment:
(optional) This function `getSampleIndexRangeForSelection` looks pretty useless atm, maybe it can be simplified by calling it directly with the `time` property?

Reply:
Yeah it's somewhat useless. It has 9 callers though so at least it's not entirely useless. I'll leave it for now, but not opposed to your suggestion.